src/state/apis/snapshot/getVotingPower.ts

```diff
@@ -26,9 +26,9 @@ export const getVotingPower = async (
   }
   const body = {
     jsonrpc: '2.0',
-    method: 'get_vp',
+    method: 'POST',
```

Comment:
just a regular rest endpoint, no need for jsonrpc/method on the body

Reply:
You're right, initially in my tests I was using the wrong endpoint and it was returning an error without it, looks fine after tests, removed in [11ddab3](https://github.com/shapeshift/web/pull/8057/commits/11ddab3fa7adcee7939f8b0c6b1d4df34827572a)